sdk/src/main/java/io/dapr/actors/Constants.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+public final class Constants {

```

Comment:
This doesn't need to be public. Only classes used by user code should be made public.

Reply:
For now, made it visible to package. If it needs to be accessed by another Java package (not jar) it will probably need to be made public again.